models/cm1/work/input.nml

```diff
@@ -16,14 +16,15 @@
   last_obs_days               = -1
   last_obs_seconds            = -1
   distributed_state           = .false.
-  ens_size                    = 20 
-  num_output_obs_members      = 20
-  num_output_state_members    = 20
+  ens_size                    = 40 
+  num_output_obs_members      = 40
+  num_output_state_members    = 40
   output_interval             = 1
   output_members              = .true.
   output_mean                 = .true.
   output_sd                   = .true.
-  stages_to_write             = 'preassim', 'analysis', 'output'
+  !stages_to_write             = 'preassim', 'analysis', 'output'
+  stages_to_write             = 'forecast', 'preassim', 'postassim', 'analysis', 'output'
   write_all_stages_at_end     = .true.
```

Comment:
All the updates to the namelist look reasonable.  Since the time I originally submitted this pull request I made additional updates that tend to optimize the performance of the DA experiments I am conducting.  Should I submit this updated namelist, or is this template good enough?